src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int side1;
+    private int side2;
+
+    public Rectangle(String color, int side1, int side2) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    public int getArea() {
```

Comment:
The method `getArea` should not start with an empty line according to the checklist.